src/web/theme/modules/Footer/Footer.scss

```diff
@@ -1,6 +1,65 @@
+$moutainHeight: 6em;
+
 .footer {
-  background: $brandPrimaryBackground;
+  background: $brandSecondaryBackground;
   color: $reverseFontColor;
   text-align: center;
   padding: 1em;
+  display: flex;
+  flex-direction: column;
+  margin-top: $moutainHeight;
+  font-family: $secondaryFontFamily;
+
+  & > div {
```

Comment:
Is the `div` really important here, or could it be any element? (hence to be replaced with `*`)

Reply:
yes it is